x-pack/plugins/security_solution/public/management/pages/event_filters/view/components/notification/index.tsx

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo, useState } from 'react';
+
+import { isCreationSuccessful, getFormEntry, getCreationError } from '../../../store/selector';
+
+import { useToasts } from '../../../../../../common/lib/kibana';
+import { useEventFiltersSelector } from '../../hooks';
+import { getCreationSuccessMessage, getCreationErrorMessage } from './translations';
+
+export const EventFiltersNotification = memo(() => {
```

Comment:
This component contains only logic and doesn't return any JSX. Looks like it should be a hook instead. What do you think about converting it to a `useEventFiltersNotification`?

Reply:
Totally agree, this is copy pasted from other component we have under management tab. Let me do this refactor and convert this into a hook